pkg/webui/lib/store/logics/status.js

```diff
@@ -115,4 +117,33 @@ const connectionCheckFailLogic = createLogic({
   },
 })
 
-export default [connectionManagementLogic, connectionCheckLogic, connectionCheckFailLogic]
+const loginStatusLogic = createLogic({
+  type: SET_LOGIN_STATUS,
+  process: ({ action }, dispatch, done) => {
+    const { isLoggedIn, sessionExpiresAt } = action.payload
+    if (!isLoggedIn || !Boolean(sessionExpiresAt)) {
+      done()
+      return
+    }
+
+    const expiryDate = new Date(sessionExpiresAt)
+    const expiryDuration = expiryDate - new Date()
+
+    if (expiryDuration >= 0 && expiryDuration < 60 * 60 * 24 * 1000) {
```

Comment:
Should this period be hard coded here ?

Reply:
It's a quite arbitrary period to prevent setting long-running timeouts. I don't mind hardcoding it here. Do you have a specific concern?